docs/source/jax_and_jaxtyping.md

```diff
@@ -40,21 +40,21 @@ for more details.
 ## Understanding JAX Arrays
 
 The main advantage of JAX over other *autodiff* libraries
-(like PyTorch or TensforFlow) is that you can pretty much use
-it as a drop-in replacement of [NumPy](https://numpy.org/),
-because almost all function from NumPy is present in
-{mod}`jax.numpy`.
+(like PyTorch or TensorFlow) is that you can pretty much use
+it as a drop-in replacement for [NumPy](https://numpy.org/),
+because JAX contains a NumPy-compatible implementation of almost
+all NumPy functions in {mod}`jax.numpy`.
 
 JAX also comes with the concept of
 [PyTrees](https://jax.readthedocs.io/en/latest/pytrees.html),
 and JAX arrays are PyTrees.
 
-The only thing your **really need to know** is that PyTrees
+The only thing you **really need to know** is that PyTrees
 are **immutable**. Hence, for compatibility with JAX's philosophy,
-DiffeRT2d's object are also **immutable PyTrees**.
+DiffeRT2d's objects are also **immutable PyTrees**.
 
-As such, every methods on those objects are likely to return
-a **new object instance**. E.g.:
+As such, every method these objects expose is likely to return
+a **new object instance**. e.g.:
```

Comment:
```suggestion
a **new object instance**. E.g.:
```

I think the capital letter should be kept here, or put a comma before instead of the dot?